.github/workflows/build_linux.yaml

```diff
@@ -122,8 +122,7 @@ jobs:
               run: |
                   npx ts-node scripts/generate-builder-config.ts \
                       ${{ steps.nightly.outputs.config-args }} \
-                      ${{ steps.debian.outputs.config-args }} \
-                      --deb-custom-control=debcontrol
```

Comment:
this is no longer needed?

Reply:
https://github.com/vector-im/element-desktop/pull/898 removed the `debcontrol` stuff, that's what originally set the deb name to `element-nightly` (whilst leaving the executable as element-desktop-nightly)